### ISSUE TITLE
SfB-Online: mePerson does not return info about logged user

### ISSUE THREAD

**Issue by ybutkevych** [OfficeDev/skype-web-sdk-samples#1]
Hi guys,

I can login into Skype for Business Online. However, I cannot retrieve any info about the logged user (email, presence, photo, etc). Do I need to add some configuration to Azure AD or this functionality is not currently supported? 

Thanks

**Comment by antonkh** [comment]
This might be due to UCWA not returning the `rel=me` resource (please check the response for `POST /applications` - it contains that `rel=me` resource with name, title and other contact details) and UCWA usually does that when the access token presented in the `Authorization` header doesn't contain the `User.ReadWrite` scope, which in turn happens when the scope isn't added in the AAD config. To check if my speculation is correct, please do the following:
1. Check if the response for `POST /applications` has the `rel=me` resource.
2. Check if the access token has the needed scopes. To do that extract the token from the `Authorization` header and decode it:

``` js
t = "ey....Yz"; // this is your token
p = t.split("."); // this gives an array of 3 elements
b = atob(p[1]); // base64 decoding
d = JSON.parse(b);
d.scp; // this is the set of scopes
```

**Comment by ybutkevych** [comment]
Thanks for your response. Indeed, there is no _User.ReadWrite_ scope added in AAD config. 

![sfb scopes](https://cloud.githubusercontent.com/assets/10098526/14178973/c8174ec4-f765-11e5-9570-03ae505987f3.png)
I can only see _Contacts.ReadWrite_, _Conversations.Initiate_ and _Meetings.ReadWrite_.

Unfortunately, there is no option to add scope _User.ReadWrite_ for Skype for Business Online in Azure 
![sfb scopes](https://cloud.githubusercontent.com/assets/10098526/14179072/3f3bb94a-f766-11e5-8681-de65e2415136.png)

Are there any ways to add this scope or it is not supportable yet?

Thanks

**Comment by sivijaya** [comment]
No, currently it is not possible to add this scope on the Azure portal.

**Comment by sidharthv** [comment]
The scope is now available on Azure (Read/Write Skype user information (preview)). With that scope selected you should get MePerson information.
